SplunkRumWorkspace/SplunkRum/SplunkRum/AppLifecycle.swift

```diff
@@ -19,6 +19,10 @@ import Foundation
 import UIKit
 import OpenTelemetrySdk
 
+let INACTIVITY_SESSION_TIMEOUT_SECONDS = 15 * 60
+private var sessionIdInActivityExpiration = Date().addingTimeInterval(TimeInterval(INACTIVITY_SESSION_TIMEOUT_SECONDS))
+var Is_New_SessionID_For_In_Activity = false
```

Comment:
Would be good to have consistent style for variables

Reply:
We create this PR from branch of APMI-2307 `https://github.com/signalfx/splunk-otel-ios/pull/51` , so this change is already implemented in it , when it will be approved and merged this problem might be solved.